### ISSUE TITLE
Issue with JSON.stringify() on to and from objects in registerRouterGuard() function

### ISSUE THREAD

**Issue by MasonLiTech** [shigeyf/vue-auth-msal#1]
Description:

In the registerRouterGuard() function, both to and from objects are serialized to JSON using JSON.stringify(). This causes issues when these objects contain circular references, which JSON.stringify() cannot handle.

In dev: everything is ok
in preview or prod: When I log in, the redirect appears to be wrong.

The problematic lines are:

```
logger.verbose(`vue-router:beforeEach():to = ${JSON.stringify(to)}`);
logger.verbose(`vue-router:beforeEach():from = ${JSON.stringify(from)}`);
```

![image](https://github.com/shigeyf/vue-auth-msal/assets/44872954/afb0da62-1df9-4090-a60d-6cc01fc8cfff)
![image](https://github.com/shigeyf/vue-auth-msal/assets/44872954/12dac620-ad3d-48bd-a734-32fac43c9d5e)

I want fix this bug, but i cant build this project.
below is the wrong when i npm run  build
![image](https://github.com/shigeyf/vue-auth-msal/assets/44872954/fcbdc83a-438a-4a58-9a89-5935ac6372d7)

**Comment by MasonLiTech** [comment]
hi @shigeyf , below is the my modified code. now is work, but i dont understand that the code is work in dev env. it seems json cannot convert.  
```
logger.verbose(`vue-router:beforeEach():to = ${JSON.stringify({ path: to.path, name: to.name })}`);
logger.verbose(`vue-router:beforeEach():from = ${JSON.stringify({ path: from.path, name: from.name })}`);
```
when i was tiring to install this proj with npm ,  it freaked out saying that some modules were missing. when i switched to using yarn, the install was successful.

**Comment by shigeyf** [comment]
@pinkpeachabc - Thanks for your issue report!!

I have now updated the module file to comment out the lines above, since these are logging lines for debugging. I released a new version: [v1.0.1](https://github.com/shigeyf/vue-auth-msal/releases/tag/v1.0.1). Please use it.
